x-pack/test/api_integration/apis/transform/delete_transforms.ts

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import expect from '@kbn/expect';
+import { TransformEndpointRequest } from '../../../../plugins/transform/common';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { COMMON_REQUEST_HEADERS } from '../../../functional/services/ml/common';
+import { USER } from '../../../functional/services/transform/security_common';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertestWithoutAuth');
+  const transform = getService('transform');
+
+  function generateDestIndex(transformId: string): string {
+    return `dest_${transformId}`;
+  }
+
+  async function createTransformJob(transformId: string, destinationIndex: string) {
+    const config = {
+      id: transformId,
+      source: { index: ['farequote-*'] },
+      pivot: {
+        group_by: { airline: { terms: { field: 'airline' } } },
+        aggregations: { '@timestamp.value_count': { value_count: { field: '@timestamp' } } },
+      },
+      dest: { index: destinationIndex },
+    };
+
+    await transform.api.createTransform(config);
+  }
+
+  describe('delete_transforms', function () {
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/farequote');
+      await transform.testResources.setKibanaTimeZoneToUTC();
+    });
+
+    after(async () => {
+      await transform.api.cleanTransformIndices();
+    });
+
+    describe('single transform deletion', function () {
+      const transformId = 'test1';
+      const destinationIndex = generateDestIndex(transformId);
+
+      beforeEach(async () => {
+        await createTransformJob(transformId, destinationIndex);
+        await transform.api.createIndices(destinationIndex);
+      });
+
+      afterEach(async () => {
+        await transform.api.deleteIndices(destinationIndex);
+      });
+
+      it('should delete transform job by transformId', async () => {
```

Comment:
Remove `job` from the description.

Reply:
Fixed in https://github.com/elastic/kibana/pull/67922/commits/d3c37b5c0d03c2ca6e211a3ff2380154223c2f0d